### PR TITLE
M5 tan FeaturedCardBlog BlogsList BlogsPage

### PR DESCRIPTION
**Pull request by Trouvaile**

### CODE REVIEW

File: src/components/modules/blogs/BlogBreadcrumb/BlogBreadcrumb.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { BreadcrumbCommon } from "src/components/common"
+import s from './BlogBreadCrumb.module.scss'
+
+const BLOG_DATA = [
+    {link: "/", name: "Home"},
+    {link: "/blogs", name: "Blog"},
+];
```
Comment (lytrankieio123) [inline src/components/modules/blogs/BlogBreadcrumb/BlogBreadcrumb.tsx:7]:
```suggestion
const BLOG_DATA = [
    {link: "/blogs", name: "Blog"},
];
```

File: src/components/modules/blogs/BlogsList/BlogsList.module.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    @apply flex flex-col spacing-horizontal items-center;
+    padding-bottom: 16.8rem;
+    .list {
+        @apply grid grid-cols-3;
+        max-width: 121.6rem;
+    }
+    .card {
+        @apply pr-8 pb-16;
+    }
+    .pagination {
```
Comment (lytrankieio123) [inline src/components/modules/blogs/BlogsList/BlogsList.module.scss:13]: check UI responsive in mobile

File: src/components/modules/blogs/BlogsList/BlogsList.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      imageSrc: image22.src,
+      title: "18",
+      description: "Aloe vera or  aloe vera  is a green plant, has thorns on the side of the skin with yellowish patches and...",
+      slug: "happy-diet"
+    },
+  
+  ]
+
+const BlogsList = ({ data = BLOGSLIST_DATA }:BlogsListProps) => {
+    const defaultBlogPageSize: number = 6;
```
Comment (lytrankieio123) [inline src/components/modules/blogs/BlogsList/BlogsList.tsx:129]: move this constant to constant.tsx (in utils), not here

File: src/components/modules/blogs/FeaturedCardBlog/FeaturedCardBlog.module.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    flex-shrink: 3;
+    max-width: 55.4rem;
+    @screen lg {
+        margin-left: 6.4rem;
+    }
+}
+.titleWrapper {
+    @apply flex flex-col items-start font-heading;
+    font-size: 3.2rem;
+    line-height: 4rem;
```
Comment (lytrankieio123) [inline src/components/modules/blogs/FeaturedCardBlog/FeaturedCardBlog.module.scss:29]: use utilities class

File: src/components/modules/blogs/BlogBreadcrumb/BlogBreadcrumb.tsx (hunk 1 above)
Comment (Trouvaile) [reply src/components/modules/blogs/BlogBreadcrumb/BlogBreadcrumb.tsx:7]: bug: Breadcrumb removed if delete {link: "/", name: "Home"}
image

**Pull request merged**